examples/plugins/tasks/attractors/README.txt

```diff
@@ -0,0 +1,12 @@
+Welcome to the attractors example.
+
+To run the application::
+
+  ``python -m attractors.run``
+
+from the ``tasks/`` directory.
```

Comment:
Maybe we need a README in `examples/plugins/tasks` in addition to OR instead of this file. Otherwise one has to navigate into the `attractors` folder to read this file, and then change directory again to run the script from outside.

Reply:
that makes sense. and i'll rename this to `index.rst` because this is one of the examples I would like shipped with the package.